Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,50 @@
-/*  Start coding below for POSITION THE CUBES TASK*/
+/*  Start coding below for POSITION THE CUBES TASK*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.parent{
+    height: 400px;
+    width: 400px;
+    background-color: #675BC6;
+}
+
+.child{
+    height: 100px;
+    width: 100px;
+}
+
+.one{
+    position: fixed;
+    border: 2px solid white;
+    top:0px;
+    left:0px;
+    background-color: #60B177;
+}
+
+.two{
+    position: fixed;
+    top: 0;
+    right: 100px;
+    border: 2px solid white;
+    background-color: #ED6D52;
+}
+
+.three{
+    place-items: center;
+    border: 2px solid white;
+    background-color:#E087E8 ;
+}
+
+.four{
+    bottom: 0px;

```

Comment:
The 'bottom' and 'top' properties should not be used together in the '.four' class. Choose one positioning property for proper alignment.